apollo-configservice/src/test/resources/data.sql

```diff
@@ -18,6 +18,9 @@ INSERT INTO App (AppId, Name, OwnerName, OwnerEmail) VALUES ('100003172','apollo
 INSERT INTO App (AppId, Name, OwnerName, OwnerEmail) VALUES ('100003173','apollo-portal','张乐','zhanglea@ctrip.com');
 INSERT INTO App (AppId, Name, OwnerName, OwnerEmail) VALUES ('fxhermesproducer','fx-hermes-producer','梁锦华','jhliang@ctrip.com');
 
+ALTER TABLE Cluster ALTER COLUMN DataChange_CreatedBy VARCHAR(255) NULL;
```

Comment:
are these alter statements still necessary as they are already defined in apollo-configservice/src/test/resources/import.sql?